app/src/components/validators.js

```diff
@@ -82,7 +83,22 @@ const models = {
         // delete tmp file
         if (tmpFile) tmpFile.removeCallback();
       }
-    }
+    },
+
+    /** @function fileConversion input/output file types must exist in fileType conversion dictionary */
+    fileConversion: (inputFileType, outputFileType) => {  
+      // get dictionary
+      const fileTypes = fileTypesComponent.fileTypesDictionary;
+      // if input/output combination not in dictionary then error
+      if(!fileTypes[inputFileType].includes(outputFileType) ){    
+        log.error('File conversion not supported');
+        return false;
+      }
+      else{
+        return true;
+      }
+    } 
```

Comment:
This would be equivalent and be able to avoid multiple return statements
``` js
    fileConversion: (inputFileType, outputFileType) => {  
      // get dictionary
      const fileTypes = fileTypesComponent.fileTypesDictionary;
      // if input/output combination not in dictionary then error
      const isSupported = fileTypes[inputFileType].includes(outputFileType);
      if (isSupported) {
        log.error('File conversion not supported');
      }
      return isSupported;
    } 
```